.circleci/config.yml

```
@@ -37,7 +37,7 @@ workflows:
       - gcp-gcr/build-and-push-image:
           name: Build and push burnham 📦
           context: data-eng-airflow-gcr
-          path: application
+          docker-context: application
           dockerfile: application/Dockerfile
```

```suggestion
          path: application
```

I think previously it was a bug that `dockerfile` didn't respect `path`, but based on the new error, these now work together as I had originally expected they would.

I think the cleanest thing to do here is to leave `path` in place, but remove the `docker-context` configuration, since the default is now doing "the right thing".